mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -225,7 +230,7 @@ class LinearRegression @Since("1.3.0") (@Since("1.3.0") override val uid: String
         summaryModel,
         model.diagInvAtWA.toArray,
         model.objectiveHistory)
-
+      instr.logSuccess(lrModel)
```

Comment:
Could you please move this to happen after setSummary?  Same for the others places below.

(The summary isn't used now, but I can imagine us wanting to log info about it when possible...in the future.)